docs/develop/contributing/index.md

```diff
@@ -4,12 +4,13 @@ title: Contributing
 permalink: /develop/contributing/
 ---
 
-To develop TSLint simply clone the repository, install dependencies and run grunt:
+To develop TSLint simply clone the repository, install dependencies:
```

Comment:
the comma is no longer required -- just say "To develop TSLint, clone the repository and install dependencies:"